app/analysis/index/controller.js

```diff
@@ -184,8 +184,20 @@ const AnalysisController = Controller.extend(AnalysisQueryParams.Mixin, {
     }
   ),
 
+  exportLimitExceeded: computed("totalItems", function() {
+    return this.get("totalItems") > this.get("exportLimit");
+  }),
+
+  exportLimitMessage: computed("exportLimit", function() {
+    return `The export limit is ${this.get(
+      "exportLimit"
+    )}. Please use filters to reduce the amount of reports.`;
```

Comment:
```suggestion
    return `The export limit is ${this.exportLimit}. Please use filters to reduce the amount of reports.`;
```

`.get` is not necessary nowadays

Reply:
should be okay now